docker-images/kafka/kafka-thirdparty-libs/2.7.x/pom.xml

```diff
@@ -24,7 +24,7 @@
     <repositories>
         <repository>
             <id>jcenter</id>
```

Comment:
Same as above

Reply:
I've made the changes and pushed in new commit. Thanks!